diff --git a/Makefile b/Makefile
index 2f905ad..348698f 100644
--- a/Makefile
+++ b/Makefile
@@ -56,7 +56,7 @@ all: ver_test ${Man} ${Bin} ${Etc}
 ver_test: ver
 	./ver
 
-${Bin}: LDFLAGS += -static -s
+${Bin}: LDFLAGS += -static
 ${Bin}: LDLIBS = -L. -lautil
 ${BinObj}: %.o: %.c ${LibUtilHdr} ${LibUtil}
 
@@ -85,7 +85,7 @@ aufs.5: aufs.in.5 c2tmac
 	chmod a-w $@
 
 c2sh c2tmac ver: CC = ${HOSTCC}
-.INTERMEDIATE: c2sh c2tmac ver
+tools: c2sh c2tmac ver
 
 Install = install -o root -g root -p
 install_sbin: File = auibusy auplink mount.aufs umount.aufs
